packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,199 @@
+# Nimble Chip
+
+## Overview
+
+The `nimble-chip` is a component that presents text, an optional "remove" button, and optional slotted content to the left of the text.
+
+### Background
+
+- [Nimble Chip component issue](https://github.com/ni/nimble/issues/413)
+- [Chip design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=9269-229459&m=dev)
+- [Spright chat feature backlog](https://github.com/ni/nimble/issues/2610) (component for attachment chips)
+
+### Containing Library
+
+The intent is for this to be put in Nimble, for the following reasons:
+
+- This is a fairly constrained component, and thus easily ported to either Angular or Blazor.
+- It has broad use possibilities.
+- Designs for the component have been completed (see above link)
+- While the remove button introduces a new interactive pattern for our controls from an accessibility perspective, the described plan seems in-line with ARIA expectations.
+
+### Non-goals
+
+- Arbitrary HTML content
+- Implementing a "chip container" component that would be responsible for laying out the chips in a particular fashion, as well as possibly managing their removal if the "remove" button on a chip is pressed. While this may be useful at some point it is not necessary for the `Chip` component to have value.
+
+### Features
+
+- Optionally removable
+    - Note that the `Chip` won't remove itself, and it will be up to the client to remove it from the DOM.
+- Allows icons to be slotted left of text content. Slotted content will be styled automatically to be appropriately sized for the chip (i.e. given a height of `controlSlimHeight`)
+- If a width is set and the label content is larger than the bounds of its container it will present with an ellipsis, and a tooltip will show the entire contents.
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+SLE keywords display:
+
+![](./spec-images/SLE-keywords.PNG)
+
+SLE axis selection:
+
+![](./spec-images/SLE-axis-selection.PNG)
+
+Nigel chat attachment chip:
+
+![](./spec-images/Nigel-attachment-chip.PNG)
+
+[Infragistics Chip component](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/inputs/chip)
+
+---
+
+## Design
+
+### API
+
+_The key elements of the component's public API surface:_
+
+- _Component Name_: `nimble-chip`
+- _Props/Attrs_:
+    - `removable` - set to show the remove button
+    - `appearance` - supports `outline` and `block` appearances
+    - `disabled` - styles the chip in the typical Nimble manner, including any slotted content, and prevents all interaction with it including being able to focus it
+- _Methods_
```

Comment:
It would probably make sense to remove the `Methods` bullet point since there aren't any methods (or explicitly put "none" under it)